plugins/modules/web_infrastructure/django_manage.py

```diff
@@ -306,7 +306,10 @@ def main():
     # these params always get tacked on the end of the command
     for param in end_of_command_params:
         if module.params[param]:
-            run_cmd_args.append(module.params[param])
+            if param in ('fixtures', 'apps'):
+                run_cmd_args.extend(module.params[param].split())
```

Comment:
Shouldn't this use `shlex.split`?

Reply:
Nice catch. Although I reckon that suffering is definitely in the path of people that creates django apps and fixtures with spaces in it.